### DISCUSSION TITLE
Checkmark option to sync fan speeds to highest based on the two set fan curves

### DISCUSSION THREAD

**Discussion by Lucid90** [seerge/g-helper#3177] [Ideas]
I do a lot of CPU intensive work most of the time and often I feel the cpu would cool faster and fans would reach lower levels if both of them did cooling work earlier in the workload. This is especially worthwhile in few seconds workloads. Also another reason working both fans would be good is because they can both stay at lower speed as opposed to the CPU fan going VROOOM and the gpu fan staying quiet. I think the biggest contributor to noise is really the speed, not the amount of air moved/amount of fans spinning.

So the idea would be to have a checkmark  that when selected will set both fans to the bigger value on the two curves it's tracking. E.G. if the CPU gets hot, the GPU fan kicks in too; if the GPU gets "relatively hotter" the CPU fan will increase to also pull some of the GPU heat through the CPU heatpipes.

**Comment by seerge** [comment]
@Lucid90 hello, G-Helper does not control fans in realtime, they are controlled by firmware. It's literally [mentioned in FAQ](https://github.com/seerge/g-helper/wiki/FAQ#how-does-g-helper-control-my-fan-speeds).

If you want to have a completely manual control you can try [Experimental build with a manual fan control](https://github.com/seerge/g-helper/discussions/2272). There you can achieve desired behavior.
